backpack/extensions/secondorder/diag_ggn/rnn.py

```diff
@@ -0,0 +1,27 @@
+"""Module implementing GGN."""
+from backpack.core.derivatives.rnn import RNNDerivatives
+from backpack.extensions.secondorder.diag_ggn.diag_ggn_base import DiagGGNBaseModule
+
+
+class DiagGGNRNN(DiagGGNBaseModule):
+    """Calculating GGN derivative."""
```

Comment:
Strictly speaking, the GGN is not a derivative but a matrix. Replace with 'diagonal'